loadsmart/datahub
# feat: [ROCK-8] Initial front-end customization

## ROCK-8

## Description

Initial front-end customization

## How to test

Run the project and check that the Loadsmart logo and colors are being used, as well as having the Loadsmart favicon and Data Catalog title on the browser's tab
